scripts/generator/generator/pyclblast.py

```diff
@@ -0,0 +1,113 @@
+
+# This file is part of the CLBlast project. The project is licensed under Apache Version 2.0. This file follows the
+# PEP8 Python style guide and uses a max-width of 120 characters per line.
+#
+# Author(s):
+#   Cedric Nugteren <www.cedricnugteren.nl>
+
+NL = "\n"
+SEPARATOR = "####################################################################################################"
+
+
+def to_np_dtype(flavour):
+    if flavour.precision_name == "S":
```

Comment:
I might have put that in a dict say like {'D': 'float64'}. Would possibly make the code a bit less redundant. The dict would have all the mappings not just the one I mentioned above.